scripts/reset_data_package_cache.py

```diff
@@ -34,7 +35,7 @@ def update_column_type_metadata(bucket: str, client):
             if subbucket == "aggregates":
                 data_package = dirs[2].split("__")[1]
             elif subbucket == "flat":
-                data_package = dirs[3].split("__")[1]
+                data_package = "__".join([dirs[3].split("__")[1], dirs[3].split("__")[2]])
```

Comment:
Doesn't need to block this PR, but seems like we have more opportunities for some sort of `Path` class to help parse these oddities. I know you added the beginnings of one a while back, but stuff like this seems like it could benefit from some abstraction too.

Reply:
yeah, i thought about this - this does use that parsing to get out the individual elements, but this is some one off behavior based on the needs of this specific data management task. If this gets repeated elsewhere I'll move it to something centralized.